docs/source/index.rst

```diff
@@ -71,3 +71,64 @@ Requirements
 * pycosat
 * pyyaml
 * requests
+
+What's new in conda 4.1.0?
```

Comment:
Let's just make this `4.1`.  We're releasing `4.1.4` today.  The "What's new" section is really about new features, and that doesn't change between `4.1.0` and `4.1.4+`.